New buffalo plugin does not build

Here's the log of what I did:

```
➜  dnnrly buffalo generate plugin github.com/dnnrly/problematic --with-gen
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/cmd/version.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/main.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/problematic/version.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/README.md
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/cmd/available.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/cmd/root.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/cmd/generate.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/genny/problematic/problematic.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/genny/problematic/options.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/genny/problematic/templates/example.txt
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/cmd/available.go
INFO[0000] /go/src/github.com/dnnrly/buffalo-problematic/LICENSE
INFO[0000] gofmt -w buffalo-problematic/cmd/available.go buffalo-problematic/cmd/generate.go buffalo-problematic/cmd/root.go buffalo-problematic/cmd/version.go buffalo-problematic/genny/problematic/options.go buffalo-problematic/genny/problematic/problematic.go buffalo-problematic/main.go buffalo-problematic/problematic/version.go
➜  dnnrly cd buffalo-problematic
➜  buffalo-problematic go get ./...
# github.com/dnnrly/buffalo-problematic/cmd
cmd/generate.go:5:2: imported and not used: "path/filepath"
cmd/generate.go:8:2: imported and not used: "github.com/gobuffalo/envy"
cmd/version.go:4:2: imported and not used: "encoding/json"
cmd/version.go:5:2: imported and not used: "os"
cmd/version.go:8:2: imported and not used: "github.com/gobuffalo/buffalo-plugins/plugins"
cmd/version.go:17:3: undefined: fmt
```

--- a/genny/plugin/templates/cmd/version.go.plush
+++ b/genny/plugin/templates/cmd/version.go.plush
@@ -1,8 +1,7 @@
 package cmd
 
 import (
-  "encoding/json"
-  "os"
+    "fmt"
 
   "<%= opts.PluginPkg %>/<%= opts.ShortName %>"
   "github.com/gobuffalo/buffalo-plugins/plugins"

--- a/genny/plugin/with/generate/templates/cmd/generate.go.plush
+++ b/genny/plugin/with/generate/templates/cmd/generate.go.plush
@@ -2,10 +2,8 @@ package cmd
 
 import (
 	"context"
-	"path/filepath"
 
 	"<%= opts.PluginPkg %>/genny/<%= opts.ShortName %>"
-	"github.com/gobuffalo/envy"
 	"github.com/gobuffalo/genny"
 	"github.com/gobuffalo/genny/movinglater/gotools"
 	"github.com/pkg/errors"

